test/code/plugins/out_oms_api_systest.rb

```diff
@@ -0,0 +1,113 @@
+require 'fluent/test'
+require_relative ENV['BASE_DIR'] + '/source/ext/fluentd/test/helper'
+require_relative ENV['BASE_DIR'] + '/source/code/plugins/oms_configuration'
+require_relative ENV['BASE_DIR'] + '/source/code/plugins/out_oms_api'
+require_relative 'omstestlib'
+
+class OutOMSApiTest < Test::Unit::TestCase
```

Comment:
A lot of duplicated code. Is it possible to share code instead?
Like a system_test class that is subclassed for every output plugin